### DISCUSSION TITLE
FindMySync updates only once

### DISCUSSION THREAD

**Discussion by sejtam** [MartinPham/FindMySync#43] [Q&A]
I am using FMS on a Mac running Sonoma, and it works (updating the locations of several airtags) only when started, or when i interact with the UI (eg requesting the status).  However, further updates are not done automatically.  

is there a way to further debug this?

**Comment by sejtam** [comment]
Also, once i place FindMySync into the background, there seems to be no way to get its UI back up.
![Screenshot 2024-10-26 at 2 23 07 PM](https://github.com/user-attachments/assets/976f5d19-becc-4893-a179-e8800b7a2a90)
![Screenshot 2024-10-26 at 2 23 14 PM](https://github.com/user-attachments/assets/25f53a97-6daa-47a8-8e81-a06d460bc463)

**Reply by bennydiamond** [reply]
Same problem here on a Proxmox VM running Sonoma.

Unless I manually press the force update button or relaunch the program, the location data is never pushed to home assistant.

Any fixes?
